src/util/SystemChannelFlags.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const BitField = require('./BitField');
+
+/**
+ * Data structure that makes it easy to interact with an {@link SystemChannelFlags#flags} bitfield.
+ * <info>Note that both welcome messages and boost messages are enabled by default,
+ * and by setting their corresponding flags you are disabling them.</info>
+ * @extends {BitField}
+ */
+class SystemChannelFlags extends BitField {}
+
+/**
+ * Numeric system channel flags. All available properties:
+ * * `WELCOME_MESSAGE_DISABLED`
+ * * `BOOST_MESSAGE_DISABLED`
```

Comment:
You should invert these, in the order of the flags (welcome message is 2, while boost messages is 1)